selenium/sample-build.js

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (C) 2021 Pixie Brix, LLC
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+const webdriver = require("selenium-webdriver");
```

Comment:
Codacy has a fix for the issue: [Do not use "require".](https://app.codacy.com/gh/pixiebrix/pixiebrix-extension/pullRequest?prid=7713940)
```suggestion
import webdriver from "selenium-webdriver";
```

Reply:
Since the script is run via "node" and is not a module, the "import" is not supported

Reply:
It can be excluded via eslintrc as part of the `overrides`

Reply:
I'll look at fixing the issues once we get a better idea of which test runner we want to use to control Selenium (e.g., Jest)